components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/Nimbus.kt

```diff
@@ -140,11 +140,22 @@ open class Nimbus(
 
     private val nimbusClient: NimbusClientInterface
 
-    override var globalUserParticipation: Boolean
-        get() = nimbusClient.getGlobalUserParticipation()
+    override var experimentsUserParticipation: Boolean
```

Comment:
If we're renaming things lets make this even better. How about just `experimentParticipation` ?